pkg/params/info/events.go

```diff
@@ -33,15 +38,32 @@ type Event struct {
 	AccountID string
 
 	// Bitbucket Server
-	CloneURL string // bitbucket server has a different cloneurl than normal url
+	CloneURL string // bitbucket server has a different url for cloning the repo than normal public html url
+	Provider *Provider
+}
 
-	ProviderToken        string
-	ProviderURL          string
-	ProviderUser         string
-	ProviderInfoFromRepo bool // whether the provider info come from the repository
+type Provider struct {
+	Token                 string
+	URL                   string
+	User                  string
+	InfoFromRepo          bool // whether the provider info come from the repository
+	WebhookSecret         string
+	WebhookSecretFromRepo bool
```

Comment:
this is never used, but this seemed useful at the time ! 🤔 

I have the feeling it will be when implementing other providers.